src/foam/nanos/ruler/Rule.js

```diff
@@ -212,7 +212,7 @@
       name: 'createdBy',
       visibility: 'RO',
       tableCellFormatter: function(value, obj) {
-        obj.userDAO.find(value).then(function(user) {
+        this.__subSubContext__.userDAO.find(value).then(function(user) {
```

Comment:
Best not to mess around with contexts directly. They're meant to be an internal thing and only used when you have no other option. Easier and better to just declare that you import userDAO.